src/tests/patient/change_events/change_events.test.js

```diff
@@ -229,10 +230,6 @@ describe('Patient Change Event Tests', () => {
             expect(messageValue2.resourceType).toBe('AuditEvent');
             expect(messageValue2.action).toBe('U');
             expect(messageValue2.agent[0].who.reference).toBe('Patient/person.81236');
-            const messageValue3 = JSON.parse(messages[2].value);
```

Comment:
I left these Observation tests here to verify the emission of the patient and proxy patient change events, but there will no longer be a final event for the Observation itself